timeseries.py

```diff
@@ -0,0 +1,74 @@
+from pandas import Series, DataFrame
+from statsmodels.tsa.stattools import adfuller
+
+
+def series_train_test_split(series: Series, split=0.7) -> Series:
+    train_size = int(len(series) * split)
+    train, test = series[0:train_size], series[train_size:]
+    return train, test
+
+
+def moving_average_smoothing(series: Series, window_size: int) -> Series:
+    """ Moving average smoothing for cleaning noisy signal """
+    return series.rolling(window=window_size).mean()
+
+
+def series_differentiation(series: Series, order: int) -> Series:
+    """
+    Timeseries differentiation.Usefull for detrend series
+    """
+    if order == 1:
+        return series.diff()
+    if order == 2:
+        return series.diff().diff()
```

Comment:
The .diff() method has a period parameter, which could be used to make the code more efficient, also easier to abstract the order variable - compare the big O notation when it comes to execution.